src/GitTfsTest/Integration/CloneTests.cs

```diff
@@ -37,7 +37,7 @@ public void ClonesEmptyProject()
                 r.Changeset(1, "Project created from template", DateTime.Parse("2012-01-01 12:12:12 -05:00"))
                     .Change(TfsChangeType.Add, TfsItemType.Folder, "$/MyProject");
             });
-            h.Run("clone", h.TfsUrl, "$/MyProject");
+            h.Run("clone", h.TfsUrl, "$/MyProject", "--initial-branch=master");
```

Comment:
Why do we need to specify the --initial-branch=master here?
My expectation would be when introducing this feature that the old existing tests don't need to be touched (or otherwise it might indicate that the fucntionality is broken if this parameter is not specified) and that a new test 
is introducd which exactly tests that this parameter is handled correctly. Now you did at the new test `CloneWithNonDefaultInitialBranch`, but why the changes here and in the following tests?

More importantly, from a quick glance over the file, it looks like we no longer test the case that this new argument is not given

Reply:
To emphasize, this comment also applies to the other changes simialar to this one in the same file ...

Reply:
This concern is now moot, because the --initial-branch option was removed from this and similar tests as of commit 6931a8e0883ee0fa829d1d72a8d9136b8353d717.

For discussion of why the change was originally made, see this [PR comment.](https://github.com/git-tfs/git-tfs/pull/1405#issuecomment-1024477958)